src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -1964,6 +1963,19 @@ void SolverInterfaceImpl::syncTimestep(double computedTimestepLength)
   }
 }
 
+void SolverInterfaceImpl::advanceCouplingSchemes()
+{
+  PRECICE_DEBUG("Advance coupling scheme");
```

Comment:
That's what TRACE would be for, no?

Reply:
This preserves the previous Debug message. I don't think that trace would be helpful here.